planet/planet.go

```diff
@@ -461,3 +468,11 @@ func injectAssetIntoMetadata(feature *geojson.Feature, asset Asset) {
 		feature.Properties["type"] = asset.Type
 	}
 }
+
+func EvalDateParam(context *Context, date string, field string) (eval bool, err error) {
```

Comment:
- "Eval" is generally a term that conventionally means "take this input and parse/evaluate it into a value". What you're probably looking for here is "Validate", as in `ValidateDataParam`. Could you do that rename for code clarity please?

- `EvalDateParam` (or `ValidateDataParam`) is a public function due to the leading capital letter, and `go-lint` requires that public functions have a preceding comment describing what they do. E.g. `// EvalDateParam takes <input> and bakes cookies with it.`